packages/core/src/BlockNoteEditor.ts

```diff
@@ -204,46 +243,45 @@ export class BlockNoteEditor {
   }
 
   /**
-   * Inserts multiple blocks before, after, or nested inside an existing block in the editor.
-   * @param blocksToInsert An array of blocks to insert.
-   * @param blockToInsertAt An existing block, marking where the new blocks should be inserted at.
-   * @param placement Determines whether the blocks should be inserted just before, just after, or nested inside the
-   * existing block.
+   * Inserts new blocks into the editor. Since `blocksToInsert` is an array of `PartialBlock` objects, some fields
+   * might not be defined. These undefined fields are assigned values from an empty paragraph block, while the `id` is
```

Comment:
what does `These undefined fields are assigned values from an empty paragraph block` mean? Is this correct, for example when creating a heading?